tests/test_reader.py

```diff
@@ -157,7 +157,7 @@ def test_container_train_property(self):
     def test_container_feature_specs_property(self):
         test_lists = [[('feature_specs', 'csv'), ('test', 'tsv')],
                       [('train', 'csv'), ('feature_specs', 'xlsx')],
-                      [('train', 'csv'), ('feature_specs', 'xls'), ('train_metadata', 'tsv')],
+                      [('train', 'csv'), ('feature_specs', 'tsv'), ('train_metadata', 'tsv')],
```

Comment:
Why "tsv" instead of "xlsx"?

Reply:
Because we are testing the combination `[('train', 'csv'), ('feature_specs', 'xlsx')]` in the previous test so I wanted to have a different combination of file type + file format.

Reply:
Got it!